src/framework/operations.hpp

```diff
@@ -33,14 +33,14 @@ namespace Operations {
 // Comparisons enum class used for Boolean function operation.
 // these are used to compare two hexadecimal strings and return a bool
 // for now we only have one comparison Equal, but others will be added
-enum class RegComparison {Equal, NotEqual, Less, LessEqual, Greater, GreaterEqual};
+enum class RegComparison {Equal, NotEqual, Less, LessEqual, Greater, GreaterEqual, Nop};
```

Comment:
`RegComparison::Nop` is not used anywhere, I think.

Reply:
Is this necessary?